pyproject.toml

```diff
@@ -1,8 +1,57 @@
 [build-system]
-requires = ["setuptools>=58.0.0", "setuptools_scm[toml]>=3.4.3"]
+requires = [
+    "setuptools>=61.2",
+    "setuptools_scm[toml]>=3.4.3"
+]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "open-prime-rando"
+description = "An open source randomizer patcher for Metroid Prime 2 and 3."
+authors = [{name = "Henrique Gemignani"}]
```

Comment:
yknow,

Reply:
you could've added your name at any moment :p